docs/testplan/DHCP-relay-stress-test.md

```diff
@@ -0,0 +1,73 @@
+# DHCP Relay Stress Test Plan
+- [Overview](#overview)
+- [Scope](#scope)
+- [Scale / Performance](#scale--performance)
+- [Related DUT CLI commands](#related-dut-cli-commands)
+- [Test structure](#test-structure)
+  - [Configuration](#configuration)
+  - [Test cases](#test-cases)
+    - [Test case](#test-case)
+
+
+
+## Overview
+
+The purpose is to test the DHCP relay service can survive the max load that we lift to CPU.
+
+### Scope
+---------
+
+CoPP is to limit the rate of traffic sent to CPU, and then generating packets above this maximum rate, the test is to ensure the DHCP relay service can handle the maximum load without failure.
+
+### Scale / Performance
+-------------------
+1. Verfy DHCP Container Status.
+2. CPU should remain within acceptable range, indicating the DUT is not overwhelmed.
+3. Check whether the function is still normal.
+4. No errors in the logs related to DHCP or CoPP.
+
+### Related **DUT** CLI commands
+----------------------------
+
+
+| **Command**                                                      | **Comment** |
+|------------------------------------------------------------------|-------------|
+| docker ps \| grep dhcp          | check the status of DHCP container            |
+| show processes cpu --verbose \| grep dhcp \| awk '{print $9}'  | check the resource utilisation   |
+| docker exec dhcp_relay supervisorctl status \| grep dhcp \| awk '{print $2}' | verify the status of processes in dhcp are running |
+| loganalyzer    | check no error keywords related to DHCP and CoPP |
+
+## Test structure 
+===============
+
+### Configuration
+-------------------
+
+1) def setup_copp_policy(): define a CoPP policy on the test device, and limit DHCP traffic to a maximum rate 600.
+2) def client_send_discover(): simulate client sending DHCPDISCOVER message from different source MAC by broadcast. Duration: 120s. e.g. PTF will send 10,000 packets per second.
+3) def client_send_request(): simulate client sending DHCPREQUEST message from different source MAC or through different interface.
```

Comment:
In common dhcp_relay scenario, we don't have DHCP server for DUT. But I found `DHCP server sends message`, is the function `server_send_xxx` is missed here?